shelley/chain-and-ledger/executable-spec/test/Examples.hs

```diff
@@ -4,51 +4,52 @@ module Examples
   ( CHAINExample(..)
   , ex1
   , ex2
+  , ex3
+  , ex4
   )
 where
 
-import qualified Data.Map.Strict as Map (empty, fromList, singleton)
+import qualified Data.Map.Strict as Map (Map, empty, fromList, singleton)
```

Comment:
a separate `import Data.Map.Strict.Map (Map)` allows for nicer type declarations, if the explicit import of `Map` is required.